html-forms/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+  <body>
+    <h1> Coding Form! </h1>
+    <form action="" method="get">
+      <label for="email">Email <br> </label>
+      <input type="email" name="email" />
+      <p>Why is coding cool?</p>
+      <select name="drop">
+        <option value="interesting">Because it is interesting!</option>
+        <option value="fun">Because it is fun!</option>
+        <option value="powerful">Because it is powerful!</option>
+      </select>
+      <p><label for="languages"> Favorite Language (choose one)</label></p>
```

Comment:
you don't want to user a `p` tag here to format because it is a text element. A div would be better!